ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -2850,8 +2850,18 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-vsphere-ipi-ovn-shared-to-local-gateway-mode-migration
-- as: vsphere-ipi-proxy-fips-f14
-  cron: 38 11 14,29 * *
+- as: vsphere-ipi-ovn-winc-p2-f14
+  cron: 32 12 8,23 * *
+  steps:
+    cluster_profile: vsphere-connected
```

Comment:
Shouldn't this profile be `cluster_profile: vsphere-8 ` to use the latest IBMC vSphere 8 cluster. I think this is the reason why the job can't locate the images.

Reply:
It is able to locate the image ..with this , checked with @jinyunma .

Reply:
@miyadav similar issue was faced by the Windows containers team today on the CI, [it turned out to be some purging problem|https://redhat-internal.slack.com/archives/C015H2WDJRY/p1690378611448979] . They suggested to undo the cluster_profile change form vsphere-connected back to vsphere as [vsphere-connected might use specific quota-slices with custom config](https://redhat-internal.slack.com/archives/CM4ERHBJS/p1690379945434649?thread_ts=1690362819.219469&cid=CM4ERHBJS), so I would suggest to revert it back.

Reply:
@jfrancoa when using cluster profile `vsphere-connected`, the job will be scheduled on ibmc vsphere which we also use for manual installation from our Jenkins. If you have already tested pass for winc template on ibmc vsphere form Jenkins, I guess it should also work well in this PR if we use cluster profile `vsphere-connected` and correct winc template.
Another concern is that we have account and permission to manage QE templates on ibmc vsphere, so suggest to run this QE winc CI job on ibmc vsphere ( using cluster profile `vsphere-connected`)

Reply:
Makes sense, I guess developers can run either on ibmc or other vsphere cluster that's why they were suggesting to revert back to vsphere. Sorry for the noise then, I was just trying to pass back the message, I wasn't sure on how the profiles did work, thanks for clarifying @jinyunma

Reply:
Thanks guys !! Is it looking good for labels now ?